istio-proxy/fetch.sh

```diff
@@ -96,7 +96,6 @@ function correct_links() {
   pushd ${CACHE_DIR}
     find . -lname '/*' -exec ksh -c '
       PWD=$(pwd)
-echo $PWD
```

Comment:
Now that I'm seeing this, I can't avoid to say: Use other variable name. PWD is kind of a reserved variable for the shell.

"Assignments to this variable may be ignored. If an application sets or unsets the value of PWD, the behaviors of the cd and pwd utilities are unspecified." -- http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html

This can be done later in other PR, of course.

Reply:
I'll update it, thanks.